src/main/resources/story/training/Task_8.story

```diff
@@ -0,0 +1,72 @@
+
+Scenario: 4.1,4.2,4.3 Add items to the shopping cart
+When Navigate to the SauceDemo website homepage
+When I fill ${swagGoodUserName},${swagPassword} to saucedemo website
+When I save text of element located by `xpath(.//*[@data-test="inventory-item-price"])->filter.index(1)` to scenario variable `min`
+When I save text of element located by `xpath(.//*[@data-test="inventory-item-price"])->filter.index(1)` to scenario variable `max`
+Given I initialize SCENARIO variable `min` with value `#{substringAfter(${min}, $)}`
+Given I initialize SCENARIO variable `max` with value `#{substringAfter(${max}, $)}`
+When I find GREATER_THAN `0` elements by `xpath(//*[@data-test="inventory-item"])` and for each element do
+|step|
+|When I save text of element located by `xpath(.//*[@data-test="inventory-item-price"])` to scenario variable `price`|
+|Given I initialize SCENARIO variable `cleanPrice` with value `#{substringAfter(${price}, $)}`                       |
+|Given I initialize SCENARIO variable `min` with value `#{eval(${cleanPrice} < ${min} ? ${cleanPrice} : ${min})}`    |
+|Given I initialize SCENARIO variable `max` with value `#{eval(${cleanPrice} > ${max} ? ${cleanPrice} : ${max})}`    |                                                                  |
```

Comment:
min and max logic is copy-pasted across 2 scenarios, please consider moving it into a composite step